pkg/operator/nfs/server.go

```diff
@@ -25,6 +25,7 @@ import (
 
 const (
 	ganeshaLog     = "/dev/stdout"
+	ganeshaPid     = "/var/run/ganesha/ganesha.pid"
```

Comment:
Is `/var/run/ganesha` in the pod volume mounts? Otherwise I would think writing to the directory could fail on openshift.

Reply:
I think Seb answered this here https://github.com/rook/rook/pull/4396#issuecomment-561057441

Reply:
> Is `/var/run/ganesha` in the pod volume mounts? Otherwise I would think writing to the directory could fail on openshift.

Now it won't fail because the container will be privileged and this is a place holder once ganesha can run with a non-root user. Sounds good?